cmd/conf/config.go

```diff
@@ -8,60 +8,37 @@ import (
 	"github.com/ethereum/go-ethereum/log"
 	"github.com/ethereum/go-ethereum/node"
 	"github.com/offchainlabs/nitro/arbnode"
+	"github.com/offchainlabs/nitro/das"
 	"github.com/pkg/errors"
 	flag "github.com/spf13/pflag"
 )
 
 const PASSWORD_NOT_SET = "PASSWORD_NOT_SET"
 
 type ConfConfig struct {
-	Dump      bool     `koanf:"dump"`
-	EnvPrefix string   `koanf:"env-prefix"`
-	File      []string `koanf:"file"`
-	S3        S3Config `koanf:"s3"`
-	String    string   `koanf:"string"`
+	Dump      bool         `koanf:"dump"`
+	EnvPrefix string       `koanf:"env-prefix"`
+	File      []string     `koanf:"file"`
+	S3        das.S3Config `koanf:"s3"`
+	String    string       `koanf:"string"`
 }
 
 func ConfConfigAddOptions(prefix string, f *flag.FlagSet) {
 	f.Bool(prefix+".dump", ConfConfigDefault.Dump, "print out currently active configuration file")
 	f.String(prefix+".env-prefix", ConfConfigDefault.EnvPrefix, "environment variables with given prefix will be loaded as configuration values")
 	f.StringSlice(prefix+".file", ConfConfigDefault.File, "name of configuration file")
-	S3ConfigAddOptions(prefix+".s3", f)
+	das.S3ConfigAddOptions(prefix+".s3", f)
 	f.String(prefix+".string", ConfConfigDefault.String, "configuration as JSON string")
 }
 
 var ConfConfigDefault = ConfConfig{
 	Dump:      false,
 	EnvPrefix: "",
 	File:      nil,
-	S3:        DefaultS3Config,
+	S3:        das.DefaultS3Config,
 	String:    "",
 }
 
-type S3Config struct {
-	AccessKey string `koanf:"access-key"`
-	Bucket    string `koanf:"bucket"`
-	ObjectKey string `koanf:"object-key"`
-	Region    string `koanf:"region"`
-	SecretKey string `koanf:"secret-key"`
-}
-
-func S3ConfigAddOptions(prefix string, f *flag.FlagSet) {
-	f.String(prefix+".access-key", DefaultS3Config.AccessKey, "S3 access key")
-	f.String(prefix+".bucket", DefaultS3Config.Bucket, "S3 bucket")
-	f.String(prefix+".object-key", DefaultS3Config.ObjectKey, "S3 object key")
-	f.String(prefix+".region", DefaultS3Config.Region, "S3 region")
-	f.String(prefix+".secret-key", DefaultS3Config.SecretKey, "S3 secret key")
-}
-
-var DefaultS3Config = S3Config{
-	AccessKey: "",
-	Bucket:    "",
-	ObjectKey: "",
-	Region:    "",
-	SecretKey: "",
-}
-
```

Comment:
This is generic s3 config which we use elsewhere, I don't think we want to move it to the das package. Is there a reason you needed to do it?